core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -473,11 +497,10 @@ import scala.util.control.NonFatal
             log.warning("Kafka commit took longer than `commit-time-warning`: {} ms", duration / 1000000L)
           }
           commitsInProgress -= 1
-          if (exception != null) reply ! Status.Failure(exception)
+          if (exception != null) sendReply(Status.Failure(exception))
           else {
-            val committed = Committed(offsets.asScala.toMap)
-            self ! committed
-            reply ! committed
+            self ! Committed(offsets.asScala.toMap)
+            sendReply(Done)
```

Comment:
Before the change senders would get back the `Committed` message as a reply. But now `Done` is sent instead? Is that deliberate?

Reply:
Very observant.
Yes, I noticed the other side doesn't care about the `Committed`

```scala
    consumerActor
      .ask(Commit(offsetsMap))(Timeout(commitTimeout))
      .map(_ => Done)
```

https://github.com/akka/alpakka-kafka/blob/docs/v1.0.1/core/src/main/scala/akka/kafka/internal/CommittableSources.scala#L114-L116